examples/readme.md

```diff
@@ -27,7 +27,7 @@ Setup - Node.js
 Setup - Python
 --------------
 
-1. `cd` into the `examples/` directory and run `pip install watson_developer_cloud flask` (or `easy_install...`) to install python dependencies
+1. `cd` into the `examples/` directory and run `pip install watson_developer_cloud flask python-dotenv pyopenssl` (or `easy_install...`) to install python dependencies.
```

Comment:
If you use the latest version of the watson sdk, do you still need `pyopenssl` or `python-dotenv`?

Reply:
python-dotenv is definitely required regardless of the SDK version, pyopenssl was required since the server was not responding properly due to encoding issues. You can see the solution which I implemented referenced here : https://github.com/samuelclay/NewsBlur/issues/642#issuecomment-275327280